bin/sidecli.re

```diff
@@ -13,17 +13,11 @@ let man = [
 
 // Todo from Andre: we may have to do peer discovery?
```

Comment:
This TODO can be removed now I guess?

Reply:
Nope, we still don't have peer discovery

Reply:
Validators will be sourced from the contract - not much to with p2p at the moment is it?

Reply:
(resolving as it is non blocking)

Reply:
yes and validators shouldn't be required on the sidecli anyway